metallb/base/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: metallb-system
+
+resources:
+  - daemonset.yaml
+  - deployment.yaml
+  - rolebinding.yaml
+  - role.yaml
+  - serviceaccount.yaml
+
+patches:
```

Comment:
Why are we patching in a base? You should reflect that directly in the base manifests instead

Reply:
The reason we're patching in a base is that when this all started the goal was to consume an upstream package without modification and then patch in the changes necessary for our environment, much like we handle RPMs by starting with the upstream source and then applying patches.

As we've discovered through this PR it's not really possible to consume an upstream package like that, so perhaps this patch is no longer necessary.